syntax = "proto2";

package AlphaDots;

message ProcessManagementRequest {
    required string model = 1;
    required int32 width = 2;
    required int32 height = 3;
    required string key = 4;
    enum Action {
        START = 0;
        STOP = 1;
    }
    required Action action = 5;
    required bool gpu = 6;
}

message ProcessManagementResponse {
    required int32 port = 1;
    required string key = 2;
}